bin/setup_dev

```diff
@@ -113,7 +113,7 @@ $PIP install --upgrade --index-url=https://pypi.python.org/simple Jinja2 \
     google-cloud-logging==1.14.0
 
 $PIP freeze
-
+deactivate
```

Comment:
can this be moved inside the subshell for the build, e.g. around line 133?  I'd be worried that this will cause problems later down the line since this deactivation is persistent and extends past the mininet build

Reply:
I think it's just easier to move mininet installation prior to venv activation.